src/renderer/lib/telemetry.js

```diff
@@ -32,15 +36,29 @@ function init (state) {
   telemetry.torrentStats = getTorrentStats(state)
   telemetry.approxNumTorrents = telemetry.torrentStats.approxCount
 
-  if (config.IS_PRODUCTION) {
-    postToServer()
-    // If the user keeps WebTorrent running for a long time, post every 12h
-    setInterval(postToServer, 12 * 3600 * 1000)
```

Comment:
Interval moved to `renderer/main.js` so that `send()` just does an instant send.